docs/src/pages/ui/primitives/select/index.page.mdx

```diff
@@ -145,17 +139,51 @@ import '@aws-amplify/ui-react/styles.css';
     <Select
       defaultValue=""
       placeholder="Please select a fruit"
-      variation="filled"
+      variation="flushed"
     >
       <option value="apple">Apple</option>
       <option value="banana">Banana</option>
       <option value="orange">Orange</option>
     </Select>
-    <Select
-      defaultValue=""
-      placeholder="Please select a fruit"
-      variation="flushed"
-    >
+  </Flex>
+</Example>
+
+### Placeholder
+
+You can set a placeholder option out of box via the `placeholder` props. The value for the placeholder option will set to empty string `""`. If you would like to pass your own placeholder with a custom value, then just simply leave it off and pass your custom one.
+
+```jsx
+import { Select } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+import { Select } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+// use the placeholder out of box
+<Select defaultValue="" placeholder="Please select a fruit">
```

Comment:
Is it necessary to always specify the defaultValue prop? I think if you leave it off it will still be an uncontrolled component, but I'm not sure if the placeholder will be selected

Reply:
The `defaultValue` is used by uncontrolled component to initialize I think. For controlled component, it should always use `value`, the `defaultValue` doesn't work. If we leave it off, then it will select the first option by default